release-notes/unreleased/2505-filter-button-too-close.yml

```
@@ -0,0 +1,7 @@
+issue_key: 2505
+show_in_stores: false
+platforms:
+  - ios
+  - android
+en: Remove unneeded padding for filter buttons
+de: Unnötiger Abstand von Poi-Filter-Buttons entfernt
```

I think this PR is more about adding padding between the filter buttons instead of removing it, isn't it?
```suggestion
en: Add padding between filter buttons.
```

---

Not really. I removed the padding around to be able to have a gap between

---

Like fine for me, I just meant that the issue was specifically about the buttons not having a gap between and therefore being right next to each other which was the problem for the users imo. If there is now more or less space around is not the relevant part, is it?

---

Margin vs padding :D